src/multihash_impl.rs

```diff
@@ -0,0 +1,106 @@
+use multihash_derive::Multihash;
+
+/// Default (cryptographically secure) Multihash implementation.
```

Comment:
maybe remove `(cryptographically secure)`, given the state of Sha1?

Reply:
Was sha1 added again? Didn't notice, @vmx and I discussed removing it and it was removed at one point

Reply:
Yes. I'll simply remove Sha1 from the default table.